src/scene/materials/standard-material.js

```diff
@@ -563,6 +60,1618 @@ class StandardMaterial extends Material {
 
     userAttributes = new Map();
 
+    /**
+     * Used to fade out materials when {@link StandardMaterial#opacityFadesSpecular} is set to false.
+     *
+     * @type {boolean}
+     */
+    alphaFade;
+
+
```

Comment:
Can you please go through and make sure only 1 line separates class field declarations please (not 2)?